llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -1468,6 +1473,14 @@ void SubtargetEmitter::emitSchedClassTables(SchedClassTables &SchedTables,
     }
     OS << "}; // " << Proc.ModelName << "SchedClasses\n";
   }
+
+  OS << "\n#if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)\n";
+  NameTable.EmitStringTableDef(OS, Target + "NameTable", /*Indent=*/"",
+                               "extern");
+  OS << "\n#else\n";
+  OS << "\nextern constexpr llvm::StringTable " << Target
```

Comment:
`extern constexpr`? I'm surprised this even compiles.

Reply:
I guess that could also be the cause of the link failure seen above